core/common/src/main/java/alluxio/security/authentication/DefaultAuthenticationServer.java

```diff
@@ -164,7 +165,8 @@ private void cleanupStaleClients() {
       }
     }
     // Unregister stale clients.
-    LOG.debug("Found {} stale channels for cleanup.", staleChannels.size());
+    LOG.debug("Found {} stale channels for cleanup:\n ->{}", staleChannels.size(),
```

Comment:
Is the formatting right? We will have a new line and then ` ->`? Also, I'm not sure if we need to list out the channels since won't we see the log of the channel getting closed from `close`? I'm just worried about the log being too verbose even at debug.

Reply:
Agreed. This will not look pretty for thousands of entries.